kphp-inspector/src/file_searcher.js

```diff
@@ -38,42 +39,139 @@ function doesFileContainFunction(fullFileName) {
 }
 
 /**
- * @param {string} q Search string from console
+ * @param {string} query Search string from console
  * @return {string[]} Full file names, to be passed to parseCppFunction()
  */
-function performSearchForFunction(q) {
-  let qParts = splitUserSearchStr(q);
-  if (!qParts.length) {
-    return [];
+function performSearchForFunction(query) {
+  const strict = query.startsWith('\\');
```

Comment:
Maybe, here and below we shouldn't use a term 'strict', describing that "strict is fqn"? Let's just use "fqn" / "is_fqn" / etc.

Reply:
Fixed.